src/components/Table/index.jsx

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './style.css';
+import StorageData from '../../utils/LocalStorage';
 
 const Table = ({ data }) => {
-  const handelDelete = () => {};
+  const handelDelete = (i) => {
+    console.log(i);
```

Comment:
when you push to GitHub u should delete error banding or debug tracer even the console log function